packages/api/src/external/commonwell/document/document-query.ts

```diff
@@ -212,6 +213,7 @@ export async function internalGetDocuments({
   const documents: Document[] = docs.flatMap(d => {
     if (d.content && d.content.masterIdentifier?.value && d.content.location) {
       return {
+        ...getExtraDocRefInfo(d.content),
```

Comment:
This is being overwritten by line 219, that sets `contained`

Reply:
It's not being overwritten because I'm returning {organization: object, practitioner: object, facilityType: string }. 219 sets the `contained` attribute.

Reply:
:truestory, I missed that.

But then, can we add those straight on the Doc Ref resource? I thought they should be on `contained`, no?